tests/unit/test_cli.py

```diff
@@ -332,6 +334,27 @@ def test_migrate_credentials_azure(ws):
         ws.storage_credentials.list.assert_called()
 
 
+def test_migrate_credentials_aws(ws, mocker):
+    mocker.patch("shutil.which", return_value=True)
+    ws.config.is_azure = False
+    ws.config.is_aws = True
+    uc_trust_policy = mocker.patch(
```

Comment:
`mocker.patch("databricks.` is prohibited. It makes tests not Easy To Change (ETC). Use constructor injection instead

See https://www.fugue.co/blog/2017-07-18-revisiting-unit-testing-and-mocking-in-python.html

Reply:
this one is tricky for cli commands, as we only have WorkspaceClient. Everything else is initialised inside the function, so we have to patch it that way, similar to https://github.com/databrickslabs/ucx/blob/2657f6c6b75165a0d91c36c0f13aa23dcd13ffd6/tests/unit/test_cli.py#L316